llmstudio/engine/providers/openai.py

```diff
@@ -48,6 +50,7 @@ async def generate_client(
                     if isinstance(request.chat_input, str)
                     else request.chat_input
                 ),
+                # tools=request.tools,
```

Comment:
so, if the client passes tools, it is ignored? shouldn't we use tools if tools are provided and functions if functions are provided and tools is None?